### DISCUSSION TITLE
Project status

### DISCUSSION THREAD

**Discussion by chrisdenman** [learnyouahaskell/learnyouahaskell.github.io#69] [General]
I've recently submitted a PR with various grammatical/spelling/... bits. I based this work on the markdown files as I presumed they were used to generate the site. As I understand things now, this is not correct in that the markdown files are not actually used.

If I redo my PR but base the changes on the files in the docs/ folder, will they be considered?

I'm also keen on completing some other PRs, adding some new docs for semigroups etc.

Thanks!

**Comment by smith558** [comment]
Hi! Most definitely. The markdown generation bit was added by a contributor but it's a bit stuck at the moment. Smaller-focused PRs are always reviewed, however.

**Reply by chrisdenman** [reply]
Great, thanks for the speedy reply. I'll get a new PR sorted and then look at the other bits that have piqued my interest.
